docs/build.md

```diff
@@ -711,44 +727,3 @@ spec:
             - Succeeded
           name: tekton-pipeline-name
 ```
-
-### Sources
-
-**Note: This feature has been deprecated, and will be removed in a future release**.
-
-Sources represent remote artifacts, as in external entities added to the build context before the actual Build starts. Therefore, you may employ `.spec.sources` to download artifacts from external repositories.
-
-```yaml
-apiVersion: shipwright.io/v1alpha1
-kind: Build
-metadata:
-  name: nodejs-ex
-spec:
-  sources:
-    - name: project-logo
-      url: https://gist.github.com/project/image.png
-```
-
-Under `.spec.sources` are the following attributes:
-
-- `.name`: represents the name of the resource, required attribute.
-- `.url`: universal resource location (URL), required attribute.
-
-When downloading artifacts, the process is executed in the same directory where the application source-code is located, by default `/workspace/source`.
-
-Additionally, we plan to keep evolving `.spec.sources` by adding more types of remote data declaration. This API field works as an extension point to support external and internal resource locations.
-
-At this initial stage, authentication is not supported; therefore, you can only download from sources without this mechanism in place.
-
-## BuildRun deletion
-
-A `Build` can automatically delete a related `BuildRun`. To enable this feature set the  `build.shipwright.io/build-run-deletion` annotation to `true` in the `Build` instance. This annotation is not present in a `Build` definition by default. See an example of how to define this annotation:
```

Comment:
As mentioned above, this is now a flag inside spec.retention. Please make sure it gets documented.